drivers/clock_control/nrf_clock_calibration.c

```diff
@@ -49,7 +50,7 @@ static struct clock_control_async_data cal_hf_on_data = {
 	.cb = cal_hf_on_callback
 };
 
-static struct device *hfclk_dev; /* Handler to hfclk device. */
+static struct device *clk_dev; /* Handler to clk device. */
 static struct device *temp_sensor; /* Handler to temperature sensor device. */
```

Comment:
These two variables are not actually handlers. I guess these comments could be just removed.

Reply:
comment removed